test/behavior/tendrl/cluster.feature

```diff
@@ -1 +1,91 @@
-Feature: CLUSTER
\ No newline at end of file
+Feature: CLUSTER
+
+  Scenario: Import a cluster
+    Given User is on "clusters" page
+     When User should see the "import" button
+      And User clicks on "import" button
+     Then User sees the "import-clusters" page
+
+  Scenario: Unmanage a cluster
+    Given User is on "clusters" page
+     When User clicks on cluster dropdown
+      And User clicks on unmanage button
+      And User sees "Unmanage Cluster" dialogue box
+      And User clicks on "unmanage" button
+     Then User should see "unmanage cluster" "task page"
+      And User closes "task page"
+      And User sees cluster is unmanaged
+ 
+  Scenario: Naming an imported cluster
+    Given User is on "clusters" page
+      And User should see the "import" button
+      And User clicks on "import" button
+      And User sees the "import-clusters" page
+  	 When User enters cluster name
+  	  And User clicks on "import" button
+  	 Then User sees "task submitted" page
+  	  And User clicks on "close" button
+  	 Then User sees the "cluster" page
+  	  And User sees name in cluster list view
```

Comment:
Maybe make cluster name generic so you can assert that what was entered is equal to what appears at the end